tutorials/ai-launchpad-producer/ai-launchpad-producer.md

```diff
@@ -0,0 +1,397 @@
+
+!![Dataset overview](img/dataset.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 7: ](Set training configuration)]
+
+With **configuration** you bind text classification scenario's executable (AI pipeline) with dataset (artifact) to train the model.
+
+1. Select **Configuration** under **ML Operations**. Click **Create**.
+
+    !![Create configuration](img/config-0.png)
+
+2. Set configuration details as shown below.
+
+    !![Create configuration](img/config-1.png)
+    <div>&nbsp;</div>
+
+    |  Field Name     | Value | Note
+    |  :------------- | :------------- | :------
+    |  Configuration Name | **`airlines-complaint-training`**   | |
+    |  Scenario           | **`text-clf-tutorial-scenario`**    | |
+    |  Version            | **`2.0.3`** or **`1.0.0`**          | Choose version similar to synced workflow |
+    |  Executable         | **`text-clf-train-tutorial-exec`**  | **Workflow executable** refers to an executable for training the model |
+
+
+3. Click **Next** when asked for **Input Parameters**.
+    > With **Input Parameters** you set values to the **placeholder for hyper-parameters** in your executable.
+
+    !![Create configuration](img/config-2.png)
+
+4. Identify your dataset row by in the **Available Artifacts** pane using the unique ID. Select the drop-down next to it and check `text-data`.
+
+    > With **Input artifacts** you set values to **placeholder for artifacts** (dataset or models) in your executable. Here `text-data` is a placeholder name for which to set value for.
+
+    !![Create configuration](img/config-3.png)
+
+5. Review your configuration settings and click **Create**.
+
+    !![Create configuration](img/config-4.png)
+
+You will see the overview of your configuration. The configuration ID is uniquely identifies your configuration.
+
+> You can create multiple **Configurations** on the same executable with different values for the placeholders, that is, **Input Artifacts** or **Parameters**.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 8: ](Start training with execution)]
+
+> ### What's an execution?
+>
+>Execution refers to the training process. It is a one-life process, therefore, you use the **configuration** to create another execution with the same settings (value bindings).
+
+1. Click **Execution** under **ML Operations**. Then click **Create**.
+
+    !![Execution Create](img/train-1.png)
+
+2. Select configuration named `airlines-complaint-training`.
+
+    Verify your configuration ID (may be different from show image) and click **Create**.
+
+    !![Execution Select Configuration](img/train-2.png)
+
+    You will be redirected to dashboard of the newly created **Execution**.
+
+2.	Observe each step of your training process.
+
+    !![Execution Create](img/train-3.png)
+
+3.	Click **Refresh**.
+
+    Once your execution status changes to `COMPLETED`, it will generate **Output Artifact** that is, model for text classification.
```

Comment:
This sentence looks odd. I think that's because of missing articles and/or words.